pkgs/applications/video/obs-studio/plugins/obs-webkitgtk.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, obs-studio
+, webkitgtk
+, glib-networking
+, meson
+, cmake
+, pkg-config
+, ninja
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "obs-webkitgtk";
+  version = "ddf230852c3c338e69b248bdf453a0630f1298a7";
```

```suggestion
  version = "unstable-2023-11-10";
```

We use the commit date prepended by unstable as version for unreleased versions.